### PR TITLE
Update web

### PR DESCRIPTION
**Pull request by makeclean**
This PR updates documentation for the Svalinn DAGMC website. It includes changes relating to
- Remove all mentions of CGM in build instructions
- Reintroduces "legacy" workflow documentation
- Introduces DAG tally Documentation
- Gives recommendations of producing DAGMC models using Tim Bohm's recommended workflow
- Add some documentation regarding watertightness

There may be some things to change, but all comments are welcome.

### CODE REVIEW

File: doc/usersguide/cubit_trelis_plugin.rst [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+--------------------------
+Obtain the `DAGMC Plug-in <http://go.wisc.edu/dagmc-trelis>`_
+
+   1.  unpack this tar file in the plugin directory of Trelis, e.g. /opt/Trelis-16.0/bin/plugins/
+   2.  make a symbolic link in the plugin directory to the dagmc plugin itself. e.g. 
+       ln -s dagmc/libdagmc_export_plugin.so .
+
+OS/X Install Instructions
+-------------------------
+Obtain the `DAGMC Plug-in <https://drive.google.com/open?id=0B_tNonFwhOqQQkdhYWxWN1drVGM>`_
```
Comment (kkiesling) [inline doc/usersguide/cubit_trelis_plugin.rst]: I don't think this link is public, so perhaps it should be moved to a spot like the linux plugin at go.wisc.edu

File: doc/usersguide/quick_start.rst [hunk 2]
```diff
@@ -1,25 +1,31 @@
...
-6. Use make_watertight, a tool that can be downloaded from the Svalinn github site, to seal your geometry.
-7. If you are using the UWUW workflow, use uwuw_preproc from the DAGMC/tools directory to mark up your model.
-8. Write the input deck for the code you want to run (you should've done this whilst you were faceting and sealing)
-9. Run the problem as you see fit
+  1. Import your model into Trelis/Cubit, assigning volumes to groups of the same material, e.g. 
+     adding all volumes that are to be steel to one group and so on
+  2. Simplify and defeature your CAD as your see fit
+  3. Imprint and merge the model
+  4. Export your model to DAGMC
+  5. Use make_watertight, a tool that is build alongside DAGMC, to seal your geometry
```
Comment (ljacobson64) [inline doc/usersguide/quick_start.rst]: this should say "built" not "build"

File: doc/usersguide/workflow/cubit_trelis_workflow.rst [hunk 3]
```diff
@@ -0,0 +1,302 @@
+Using Trelis/Cubit for the DAGMC Workflow
+-----------------------
+
+The general workflow for the production of models for analysis using DAGMC
+looks like that show in the Figure below.
```
Comment (ljacobson64) [inline doc/usersguide/workflow/cubit_trelis_workflow.rst]: should be "shown" not "show"

File: doc/usersguide/workflow/cubit_trelis_workflow.rst [hunk 4]
```diff
@@ -0,0 +1,302 @@
...
+     Check and possibly remove small features (curves, surfaces, vols)
+        - small areas (check "hydraulic" length and "regular" length)
+         **group "smallsurfaces" add surface with area < 1.e-3**
+        - small curves (check "hydraulic" length and "regular" length)
+         **group "smallcurves" add curve with length <1.e-4**
+        - small volumes
+         volofvolcubit.py-examine small volumes (vol<=0.0000 might need to autoheal)
+4. create pre-imprint/merge table of volume of volumes (use volofvolcubit.py)
+   Imprint model **imprint body all**
+5. Merge model
```
Comment (ljacobson64) [inline doc/usersguide/workflow/cubit_trelis_workflow.rst]: command is **merge all**

File: doc/usersguide/quick_start.rst [hunk 5]
```diff
@@ -1,25 +1,31 @@
...
-3. ``CRITICAL``` Imprint and merge the model.
-4. Export your model as an ACIS file; remember to check the export attributes box in the save window.
-5. Use dagmc_preproc, a tool found in the moab/bin directory, to produce a faceted representation of the CAD model.
-6. Use make_watertight, a tool that can be downloaded from the Svalinn github site, to seal your geometry.
-7. If you are using the UWUW workflow, use uwuw_preproc from the DAGMC/tools directory to mark up your model.
-8. Write the input deck for the code you want to run (you should've done this whilst you were faceting and sealing)
-9. Run the problem as you see fit
+  1. Import your model into Trelis/Cubit, assigning volumes to groups of the same material, e.g. 
+     adding all volumes that are to be steel to one group and so on
+  2. Simplify and defeature your CAD as your see fit
```
Comment (kkiesling) [inline doc/usersguide/quick_start.rst]: "as _you_ see fit"

**Comment by makeclean** [comment]
Ok I've just added some new commits and moved the gallery around so it appears on the main page, I think I'm pretty happy with this, i would appreciate if someone could build the docs and see what they think. All comments are welcome, I would like to get this live asap. How about @ljacobson64 could you take another look?

**Comment by ljacobson64** [comment]
So the slideshow looks great, but it makes it impossible to read anything below it while it's running because all the images have a different height.

**Comment by ljacobson64** [comment]
I think that either someone should update the Upcoming Features page or we should remove the link to it, because right now it's pretty bare-bones and I don't think it gives a very good idea of the upcoming features we're actually working on.

**Comment by ljacobson64** [comment]
I see a few references to something called "AutomatedCubitConversion" in multiple pages but I don't know what that is and I can't find any information on it anywhere. My instinct is that it probably isn't something we use anymore.

**Comment by Pshriwise** [comment]
@ljacobson64 re:slideshow

Yeah that implementation of the slideshow wasn't really meant to be thrown into the middle of a bunch of text. I'll see what I can do about maintaining the image size and PR into this branch

**Pull request merged**